### PR TITLE
Add render test for SurfaceLayers

### PR DESCRIPTION
**Pull request by txanatan**

### CODE REVIEW

File: tests/test_layers.py [hunk 1]
```diff
@@ -33,3 +33,44 @@ def test_sizes(self):
...
+        test_surface.blit(top_left, (0, 0))
+        test_surface.blit(top_right, (subsurface_size[0], 0))
+        test_surface.blit(bottom_left, (0, subsurface_size[1]))
+
+        # Create the SurfaceLayers object and fill it with our layers
+        surface_layers = sappho.SurfaceLayers(self.target_surface, 4)
+        surface_layers[0].blit(background, (0, 0))
+        surface_layers[1].blit(top_left, (0, 0))
+        surface_layers[2].blit(top_right, (subsurface_size[0], 0))
+        surface_layers[3].blit(bottom_left, (0, subsurface_size[1]))
```
Comment (lillian-lemmer) [inline tests/test_layers.py]: Maybe it'd be better to have the first layer be a canvas be this color instead of filling in the bottom right quarter?

**Comment by txanatan** [comment]
@lillian-lemmer is this good to merge?

**Comment by txanatan** [comment]
For reference, the test renders this:

test

**Comment by lillian-lemmer** [comment]
:+1: Magnificent work, @txanatan! This is _really_ cool! Yes, go ahead and merge to `next-release` if you'd like. :D

**Pull request closed without merging**